### ISSUE TITLE
Udacity: Deep Learning: Neural networks

### ISSUE THREAD

**Issue by mbhushan** [mbhushan/ml#10]
Gradient Descent Algorithm.
<img width="1035" alt="screen shot 2018-02-10 at 5 49 55 am" src="https://user-images.githubusercontent.com/898047/36062692-55ee0174-0e26-11e8-907b-bb70b3f36cfe.png">
